crates/primitives/src/basefee.rs

```diff
@@ -1,11 +1,13 @@
 //! Helpers for working with EIP-1559 base fee
 
-use crate::constants;
-
 /// Calculate base fee for next block. [EIP-1559](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1559.md) spec
-pub fn calculate_next_block_base_fee(gas_used: u64, gas_limit: u64, base_fee: u64) -> u64 {
-    let gas_target = gas_limit / constants::EIP1559_ELASTICITY_MULTIPLIER;
-
+pub fn calculate_next_block_base_fee(
+    gas_used: u64,
+    gas_limit: u64,
+    base_fee: u64,
+    base_fee_params: &crate::BaseFeeParams,
```

Comment:
they can be passed as copy
```suggestion
    base_fee_params: crate::BaseFeeParams,
```

Reply:
done, also changed next_block_base_fee to accept copy instead of reference.